addons/popochiu/editor/popups/create_room/create_room.gd

```diff
@@ -117,7 +117,8 @@ func _create() -> void:
 	
 	# ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 	# Create the room instance
-	var new_room: PopochiuRoom = preload(BASE_ROOM_PATH).instantiate()
+	var new_room: PopochiuRoom = load(BASE_ROOM_PATH).instantiate()
```

Comment:
Can you explain like I'm 5 what's the deal between load and preload in this context? I can't fathom the impacts.

Reply:
`load` tries to load the resource during runtime, `preload` does it while in the Editor. Looks like changes in Godot 4 make preloading resources behave different than in Godot 3.x. Then, preloading this audio cues here was the reason why calling something like `A.sfx_xxx.play()` didn't work. I don't know the details of that change.

Reply:
Strange. I will dig deeper into this, seems quite impactful and important to understand.
Thanks